jasmine/spec/inverted-index-test.js

```diff
@@ -0,0 +1,71 @@
+const books = require('./books.json')
+
+
+describe('InvertedIndex Class', () => {
+  beforeEach(() => {
+    this.invertedIndex = new InvertedIndex();
+    this.index = this.invertedIndex.createIndex('books', books);
+  });
+
+  describe('Constructor', () => {
+    it('can create inverted index instance', () => {
+      expect(typeof this.invertedIndex).toEqual('object');
+      expect(this.invertedIndex instanceof InvertedIndex).toBe(true);
+    });
+
+    it('has an indexes object to hold all indexes', () => {
+      expect(typeof this.invertedIndex.indexes).toEqual('object');
+    });
+  });
+
+ describe('GetWords', () => {
+    it('should return an array of words', () => {
+      expect(InvertedIndex.getWords(books[0].title))
+        .toEqual(['alice', 'in', 'wonderland']);
+    });
+
+    it('filters out symbols', () => {
+      expect(InvertedIndex.getWords('alice # in* Won@derland'))
+        .toEqual(['alice', 'in', 'wonderland']);
+    });
+  });
+
+  describe('CreateIndex', () => {
+    it('creates an index', () => {
+      expect(this.index.a).toEqual([0, 1, 2]);
+      expect(this.index.alice).toEqual([0]);
+    });
+  });
+
+
+  describe('GetIndex', () => {
+    it('should return a particular index', () => {
+      const index = this.invertedIndex.getIndex('books');
```

Comment:
You have just one index in the entire test. How do we know the function would get the correct index for each of them.

Try to test that and also test the exact result of the index, not just for a few words in here.

Also, you should test that an index is not returned when no book with said name was uploaded.